doc/nrf-bm/migration/nrf5_bm_migration.rst

```diff
@@ -223,34 +244,55 @@ This is due to several factors, including:
-The ``MBR`` acted as a very simple first-stage bootloader, only booting the application or the bootloader, and supporting basic copy functionality.
-The nRF5 bootloader had the capability to download new firmware, or SoftDevice + Bootloader images, which would then be verified and copied by the ``MBR``.
+The firmware loader in |BMshort| is somewhat comparable to the nRF5 Bootloader in functionality.
+However, their roles differ slightly.
+While the nRF5 Bootloader managed both the download process and image validation, the firmware loader in |BMshort| is solely responsible for the download process.
+Image validation is handled by MCUboot.
 
-The bootloader has changed - |BMshort| uses the open-source MCUboot project as first-stage (immutable) bootloader, instead of the MBR.
-MCUboot is also used in the |NCS| and other open-source projects.
+The process for updating the SoftDevice and the firmware loader is similar in both the old and new solutions, requiring the reuse of application space as storage for the update.
+The key difference lies in how the update is moved from temporary storage to the correct partition.
+In the nRF5 SDK, this copy functionality was managed by the MBR, whereas in |BMshort|, it is handled by an installer image that is downloaded along with the new update.
 
-A major difference is that the nRF5 bootloader included the transport (such as BLE, UART), whereas MCUboot does not.
-MCUboot just decides which firmware to boot and verifies it before booting.
+Protocol Changes
+================
 
-The actual download of the new firmware image is done by a dedicated firmware image called the Firmware Loader.
-This firmware is provided in |Bmshort|.
-In case of an application update, the Firmware Loader copies the new firmware in the application bank (or slot).
-MCUboot will then verify and boot the updated firmware.
+The protocol used to upload new images to the device has also changed.
+The previous nRF5 solution utilized a Nordic proprietary DFU protocol, while |BMshort| adopts the `MCUmgr SMP protocol`_, which is also used in the |NCS|.
 
-In case of a SoftDevice or Firmware Loader application update, the Firmware Loader on the device receives an image called Installer which is bundled with the new SoftDevice and/or Firmware Loader application.
-The Installer firmware is also provided in |BMshort|.
-This image is saved in place of the application by the Firmware Loader.
-The Installer boots next after being verified by MCUboot, and proceeds to overwrite the Firmware Loader and SoftDevice as necessary.
-The new Firmware Loader boots next to receive a new application image.
+Tool Compatibility
+==================
 
-The host and mobile tools for DFU have changed, and new versions of both are made available by Nordic.
+The PC and mobile tools available for the |NCS| are also compatible with the |BMshort| solution.
```

Comment:
I think that `desktop` is better because it includes also MacOS (`PC` is commonly used to refer to Windows machines).
```suggestion
The desktop and mobile tools available for the |NCS| are also compatible with the |BMshort| solution.
```